dev/io.openliberty.org.apache.myfaces.3.0/src/org/apache/myfaces/config/FacesConfigurator.java

```diff
@@ -503,100 +481,61 @@ public void update()
                     application.publishEvent(facesContext, PostConstructApplicationEvent.class,
                             Application.class, application);
                 }
+                catch (Exception e)
+                {
+                    log.severe("Error during configuration clean-up" + e.getMessage());
+
+                    // We still want to update the timestamp to avoid running purge on every subsequent
+                    // request after this one.
+                    //
+                    lastUpdate = System.currentTimeMillis();
+                }
             }
         }
     }
 
-    private boolean purgeConfiguration() throws NoSuchMethodException, IllegalAccessException, InvocationTargetException
+    private void purgeConfiguration()
     {
-
-        Method appFactoryPurgeMethod;
-        Method renderKitPurgeMethod;
-        Method lifecyclePurgeMethod;
-        Method facesContextPurgeMethod;
-
-        System.out.println("PURGE");
```

Comment:
I didn't see this upstream so I'm assuming you just had this here for initial testing locally?

Reply:
Oops. Yes, it was for testing